WooCommerce/Classes/ViewRelated/Orders/Cells/FulfillButtonTableViewCell.swift

```diff
@@ -3,10 +3,8 @@ import UIKit
 
 /// Displays the list of Products associated to an Order.
 ///
-class ProductListTableViewCell: UITableViewCell {
-    @IBOutlet public var verticalStackView: UIStackView!
+class FulfillButtonTableViewCell: UITableViewCell {
```

Comment:
Would you consider making it final?

Reply:
Oops, yes. Once a rule is in place, it's perfectly acceptable to demand it gets followed. "Add final" is a legitimate code review comment in my opinion.